dashboard/modules/log/log_manager.py

```diff
@@ -215,6 +233,13 @@ async def resolve_filename(
                 resolving by other ids'. Default to "out".
             submission_id: The submission id for a submission job.
         """
+        start_offset = None
+        end_offset = None
+        if suffix not in ["out", "err"]:
```

Comment:
Is it applied only for task logs? Otherwise, there are files that have .txt, .log, or any custom profiling files users want to download, so we shouldn't have this check?

Reply:
yeah, it's only used for worker/actor/task log now. But the logic is kind of implicit - so I can make the check there.

Reply:
Yeah let's make sure the API itself is general enough.